llvm/include/llvm/DebugInfo/DWARF/DWARFAcceleratorTable.h

```diff
@@ -543,6 +543,19 @@ class DWARFDebugNames : public DWARFAcceleratorTable {
       return StrData.getCStr(&Off);
     }
 
+    /// Compares the name of this entry against Target, returning true if they
+    /// are equal. This is helpful is hot code paths that do not need the length
+    /// of the name.
+    bool sameNameAs(StringRef Target) const {
```

Comment:
Is this going to be used elsewhere? If not maybe reduce scope and move to cpp as static?

Reply:
Its implementation requires `private` information, and I was hoping to add unittests for this (and everything in this class) functionality once https://github.com/llvm/llvm-project/pull/79666 gets merged